apps/admin-x-activitypub/src/api/activitypub.test.ts

```diff
@@ -1287,7 +997,7 @@ describe('ActivityPubAPI', function () {
                 fakeFetch
             );
 
-            const actual = await api.getFeed();
+            const actual = await (api[methodName as keyof typeof api] as (next?: string) => Promise<GetFeedResponse>)();
```

Comment:
Is `GetFeedResponse` common across all of the paginated endpoints? This type was specific for the return value of the `feed` endpoint

Reply:
Right, I reused it to make it work initially. But need to clean this.. the profile has exact same response. So I'll rename this to something like `PaginatedPostsResponse` and reuse it.